Apps/Database/src/Context/GatewayDbContext.cs

```diff
@@ -427,6 +428,24 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .Property(e => e.Code)
                 .HasConversion(auditGroupCodeConverter);
 
+            // Create Foreign Key for DelegateInvitation
+            modelBuilder.Entity<DelegateInvitation>()
+                .HasOne(c => c.ResourceDelegate)
+                .WithMany(p => p.DelegateInvitations)
+                .HasForeignKey(c => new { c.ResourceOwnerHdid, c.ProfileHdid, c.ReasonCode });
+
+            ValueConverter<DelegateInvitationStatus, string> delegateInvitationStatusConverter = new(
+                v => EnumUtility.ToEnumString(v, true),
+                v => EnumUtility.ToEnum<DelegateInvitationStatus>(v, true));
+
+            modelBuilder.Entity<DelegateInvitation>()
+                .Property(e => e.Status)
+                .HasConversion(delegateInvitationStatusConverter);
+
+            modelBuilder.Entity<DelegateInvitationStatusCode>()
+                .Property(e => e.Code)
+                .HasConversion(delegateInvitationStatusConverter);
```

Comment:
The relationship between DelegateInvitation and DelegateInvitationStatusCode also needs to be established.

e.g.
```
            modelBuilder.Entity<DelegateInvitation>()
                .HasOne<DelegateInvitationStatusCode>()
                .WithMany()
                .HasPrincipalKey(k => k.Code)
                .HasForeignKey(k => k.Status)
                .OnDelete(DeleteBehavior.Restrict);
```

Reply:
Added.